imaging/Pixelator.cpp

```diff
@@ -48,6 +49,18 @@ namespace imaging
 		return outputStream.str();
 	}
 
+	bool Pixelator::display(const std::string& in_imageName)

```

Comment:
is that still needed? OTOH, the whole shell approach needs a review, so this can be tackled from there, too.